src/libraries/BytesLib.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+
+/*
+ * @title Solidity Bytes Arrays Utils
+ * @author Gonçalo Sá <goncalo.sa@consensys.net>
+ *
+ * @dev Bytes tightly packed arrays utility library for ethereum contracts written in Solidity.
+ *      The library lets you concatenate, slice and type cast bytes arrays both in memory and storage.
+ */
+
+pragma solidity ^0.8.0;
```

Comment:
Add additional comment and link to the token-bridge-contracts file

Reply:
https://github.com/OffchainLabs/nitro-contracts/pull/65/commits/257476acbb097bc73ecb70855bdefee5d82199ee